### PR TITLE
PI-405: Package upgrades - JS and Python

### PR DESCRIPTION
**Pull request by Hesbon5600**
## Ticket

Upgrade the following packages:

**JS**
- micromatch

**Python**
- cookiecutter

I have successfully run the module installation command

```bash
$ node index.js create --name simple-django-module --type django 
$ node index.js add django-simple-django-module --project test-project
```
image

[PI-405](https://crowdbotics.atlassian.net/browse/PI-405)

_Related tickets:_
_Related PRs:_

## Type of PR

- [ ] Bugfix
- [ ] New feature
- [ ] Minor changes

## Changes introduced

_Describe the changes being introduced in this PR_
_Include screenshots if necessary_
_Please link any documentation reference that relate to the changes_

## Test and review

_Describe how a reviewer should test your PR_

[PI-405]: https://crowdbotics.atlassian.net/browse/PI-405?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1969,17 +1969,17 @@ argparse@^2.0.1:
 
 arr-diff@^4.0.0:
   version "4.0.0"
-  resolved "https://registry.npmjs.org/arr-diff/-/arr-diff-4.0.0.tgz"
+  resolved "https://registry.yarnpkg.com/arr-diff/-/arr-diff-4.0.0.tgz#d6461074febfec71e7e15235761a329a5dc7c520"
```
Comment (jbrendel) [inline yarn.lock:1972]: What was the reason for this switch? From `npmjs.org` to `yarnpkg.org` ?

Comment (Hesbon5600) [reply yarn.lock:1972]: This was autogenerated when running `yarn upgrade "package"`. I didn't change it explicitly.

Comment (jbrendel) [reply yarn.lock:1972]: Not saying this is wrong, but we are changing something all over the place and this is odd.

There is this interesting SO thread here: https://stackoverflow.com/questions/69526949/yarn-lock-file-resource-source-changed-from-registry-yarnpkg-com-to-registry-npm

(even though in their case it went the other way around)

Looks like a single import/install done differently can change all entries in the lock file.

What are we using in other repos? Is it mostly npmjs.org, or yarnpkg.org?

Comment (Hesbon5600) [reply yarn.lock:1972]: Most repos use `yarnpkg.org`. @alaisgomes might be the best person to tell us if there is any risk with the swirtch.

**Review by jbrendel — APPROVED** [review]

**Pull request merged**